<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial Master</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>200</width>
      <height>240</height>
     </rect>
    </property>
    <property name="title">
     <string>串口配置</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>68</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>串口号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>68</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>波特率：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>68</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>数据位：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>68</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>校验位：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>68</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>停止位：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxPort">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxBaud">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxData">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxParity">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxStop">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>190</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>200</width>
      <height>230</height>
     </rect>
    </property>
    <property name="title">
     <string>接收配置</string>
    </property>
    <widget class="QCheckBox" name="checkBoxRecieve">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>十六进制显示</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumberRecieve">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>65</y>
       <width>160</width>
       <height>30</height>
      </rect>
     </property>
     <property name="digitCount">
      <number>9</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonClearCounterRecieve">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>105</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>清除计数</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonClearDataRecieve">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>105</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>清除数据</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>200</width>
      <height>230</height>
     </rect>
    </property>
    <property name="title">
     <string>发送配置</string>
    </property>
    <widget class="QCheckBox" name="checkBoxSend">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>十六进制发送</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxPoll">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>自动循环发送</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxInterval">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>190</y>
       <width>90</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>30</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>间隔</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>190</y>
       <width>30</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>毫秒</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumberSend">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>65</y>
       <width>161</width>
       <height>30</height>
      </rect>
     </property>
     <property name="digitCount">
      <number>9</number>
     </property>
     <property name="mode">
      <enum>QLCDNumber::Dec</enum>
     </property>
     <property name="value" stdset="0">
      <double>0.000000000000000</double>
     </property>
     <property name="intValue" stdset="0">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonClearCounterSend">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>105</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>清除计数</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonClearDataSend">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>105</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>清除数据</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxMessage">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>500</y>
      <width>770</width>
      <height>230</height>
     </rect>
    </property>
    <property name="title">
     <string>信息发送</string>
    </property>
    <widget class="QCheckBox" name="checkBoxSendEnable1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>消息1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSendEnable2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>消息2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSendEnable3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>消息3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSendEnable4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>消息4</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSendEnable5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>消息5</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSend2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>65</y>
       <width>550</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSend3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>105</y>
       <width>550</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSend4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>145</y>
       <width>550</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSend5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>185</y>
       <width>550</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSend1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>25</y>
       <width>550</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSend1">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>25</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSend2">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>65</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSend3">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>105</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSend4">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>145</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSend5">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>185</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>770</width>
      <height>230</height>
     </rect>
    </property>
    <property name="title">
     <string>信息接收</string>
    </property>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>20</y>
       <width>761</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>771</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>动态曲线</string>
    </property>
    <widget class="QChartView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>20</y>
       <width>761</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="mainMenubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionClose"/>
    <addaction name="actionSave"/>
    <addaction name="actionRefresh"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>窗口(&amp;W)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionClose"/>
   <addaction name="actionSave"/>
   <addaction name="actionRefresh"/>
  </widget>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/open.png</normaloff>:/images/images/open.png</iconset>
   </property>
   <property name="text">
    <string>打开串口</string>
   </property>
   <property name="toolTip">
    <string>建立连接</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/close.png</normaloff>:/images/images/close.png</iconset>
   </property>
   <property name="text">
    <string>关闭串口</string>
   </property>
   <property name="toolTip">
    <string>断开连接</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/save.png</normaloff>:/images/images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存消息</string>
   </property>
   <property name="toolTip">
    <string>保存消息窗的数据</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/refresh.png</normaloff>:/images/images/refresh.png</iconset>
   </property>
   <property name="text">
    <string>刷新串口</string>
   </property>
   <property name="toolTip">
    <string>刷新串口列表</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/exit.png</normaloff>:/images/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
